eZ/Publish/Core/Base/Tests/PHPUnit5CompatTrait.php

```diff
@@ -9,6 +9,8 @@
 
 /**
  * Trait for PHPUnit 5 Forward Compatibility, for PHPUnit 4.8 use and up.
+ *
+ * @deprecated since 7.1, will be removed in 8.0. We are using PHPUnit 6, so this trait is obsolete.
```

Comment:
Only one nitpick: Trait can technically be removed once support for 6.7 ends, to avoid issues on merges. Could perhaps mention something like that so it is more clear for everyone.

Reply:
Updated in https://github.com/ezsystems/ezpublish-kernel/pull/2240/commits/9e1dd4e8bd1672dd37428e12113c442b28e9a606

Reply:
> to avoid issues on merges

Shouldn't issues happen only the first time the branch is merged? After that, it should be fine, right?

Reply:
yes, but it will happen for any PR introducing new test which adds use of the trait.

Reply:
True, yes :)